api/src/main/java/org/openmrs/module/fhir2/api/FhirTaskService.java

```diff
@@ -20,15 +20,15 @@
 /**
  * Contains methods pertaining to creating/updating/voiding Tasks
  */
-public interface FhirTaskService {
+public interface FhirTaskService extends FhirService<Task> {
 	
 	/**
 	 * Get task by the UUID
 	 * 
 	 * @param uuid
 	 * @return task with given internal identifier
 	 */
-	Task getTaskByUuid(String uuid);
+	Task getTask(String uuid);
```

Comment:
This should actually just be renamed to `get()` to be consistent with the `FhirService` API.

Reply:
Note that the other methods, like `saveTask()` and `updateTask()` should also be updated to `save()` and `update()`.